jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcTenantMigrationRepository.java

```diff
@@ -0,0 +1,77 @@
+package io.kestra.jdbc.repository;
+
+import io.kestra.core.repositories.TenantMigrationInterface;
+import io.kestra.jdbc.JooqDSLContextWrapper;
+import java.util.List;
+import org.jooq.Condition;
+import org.jooq.DSLContext;
+import org.jooq.Field;
+import org.jooq.Table;
+import org.jooq.impl.DSL;
+
+public abstract class AbstractJdbcTenantMigrationRepository implements TenantMigrationInterface {
```

Comment:
```suggestion
public abstract class AbstractJdbcTenantMigration implements TenantMigrationInterface {
```

It's not a repository as it didn't manage any persisted resources.